packages/v4/src/content/design-guidelines/components/button/button.md

```diff
@@ -99,6 +99,22 @@ Call To Action buttons and links are used on websites to direct users to complet
 
 Call To Actions have similar variants to regular buttons (although they have a larger padding), and follow the same visual hierarchy. However, CTA buttons are used only to indicate a desired next step, whereas other button types may also be used to link to additional information or actions. Call To Actions should be used sparingly (ideally one per page, or a primary and secondary CTA).
 
+#### Progress button
+
+A progress button can be used to provide the user with feedback that an action is in progress after the button is clicked. To indicate that an action is taking place, a spinner is inserted in front of the button label. You can optionally replace the spinner with a completion icon when the action completes.
+
+_Example of using a progress button_
+
+<img src="./img/progress-button.png" alt="Example of progress button usage" width="1435"/>
+
+1. The user clicks the submit button.
+
+2. A spinner indicates that the operation is in progress.
+
+3. When complete, use a success icon (fa-check-circle) and a change in the label text to indicate that the operation was successfully completed.
+
+Use a progress button when clicking a submit button will not navigate the user to a new page or open a modal. For operations that will take more than a few seconds, use of a modal with a progress bar is recommended to reflect status and indicate time to completion.
```

Comment:
All looks great, my only comment is I would move this to be above the image, with the rest of the general guidelines around progress buttons!